src/OSPSuite.Core/Domain/PKAnalyses/DynamicPKParameter.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System.Linq;
+using OSPSuite.Core.Domain.Services;
+
+namespace OSPSuite.Core.Domain.PKAnalyses
+{
+   public enum StandardPKParameter
+   {
+      Unknown,
+      Cmax,
+      Cmin,
+      Tmax,
+      Tmin,
+      CTrough,
+      Auc,
+      Aucm,
+      AucInf,
+      AucTendInf,
+      Mrt,
+      FractionAucEndToInf,
+      Thalf,
+      Vss,
+      Vd,
+   }
+
+   public class DynamicPKParameter : PKParameter
+   {
+      // Parameter such as AUC, Cmax, Tmax that will be used to calculate the parameter
+      public StandardPKParameter StandardPKParameter { get; set; } = StandardPKParameter.Unknown;
+
+      public float? StartTime { get; set; }
+      public float? EndTime { get; set; }
+
+      /// <summary>
+      ///    1-based index of the application that should be used to estimate the start of the interval
+      /// </summary>
+      public int? StartApplication { get; set; }
+
+      /// <summary>
+      ///    1-based index of the application that should be used to estimate the end of the interval
+      /// </summary>
+      public int? EndApplication { get; set; }
+
+      /// <summary>
+      ///    Time offset in min to apply to the start time (either from the absolute <see cref="StartTime" /> or the start time
+      ///    estimated from the application <see cref="StartApplication" />
+      /// </summary>
+      public float? StartTimeOffset { get; set; }
+
+      /// <summary>
+      ///    Time offset in min to apply to the start time (either from the absolute <see cref="EndTime" /> or the start time
+      ///    estimated from the application <see cref="EndApplication" />
+      /// </summary>
+      public float? EndTimeOffset { get; set; }
+      
+      /// <summary>
+      /// Dose parameter that will be used to calculate a normalized parameter. Not that if specified, only the normalized parameter will be added
+      /// </summary>
+      public double? DoseForNormalization { get; set; }
```

Comment:
This can be anything. Dose, DosePerBW etc... this will be set by the caller and it is its reponsabilioty to ensure that this makes sense with the Dimension as we have no way to know what this parameter is